authors/settings.py

```diff
@@ -19,11 +20,18 @@
 # See https://docs.djangoproject.com/en/1.11/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = '7pgozr2jn7zs_o%i8id6=rddie!*0f0qy3$oy$(8231i^4*@u3'
+SECRET_KEY = os.environ.get('SECRET_KEY') 
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
-
+DEBUG = os.environ.get('IS_DEBUG_ON')
```

Comment:
I like the fact that you remembered to not leave Debug mode on in deployment environment. I have manually tested following your steps and the credentials you shared on slack. I have confirmed it is successfully deployed.

Reply:
thanks @HabibSentongo + the timely review